index.bs

```diff
@@ -7992,9 +7989,9 @@ a reference to the same object that the IDL value represents.
         starting at byte offset |offset|.
 </div>
 
-<div algorithm="detach an arraybuffer">
+<div algorithm>
```

Comment:
@tabatkins can tell you more here, but my understanding is that, for now, they're only used to give helpful error messages. I think the only thing they check at present is variables which are defined but not used. Incidentally, that's how we picked up a bunch of copy/paste errors when we converted to using the algorithm construct.